README.md

```diff
@@ -317,6 +317,50 @@ export default withStyles(({ color, unit }) => ({
 
 `className` and `style` props must not be used on the same elements as `css()`.
 
+#### Example with React-Router's `Link`
+[React-Router][react-router]'s [`<Link/>`][react-router-link] and `<IndexLink/>` components accept `activeClassName='...'` and `activeStyle={{...}}` as props. To mimic this functionality you can use React-Router's [`withRouter()`][react-router-with-router] HOC on top of `withStyles()`:
+
+```jsx
+import React from 'react';
+import { withRouter, IndexLink, Link } from 'react-router';
+import { css, withStyles } from '../withStyles';
+
+function Nav ({ router, styles }) {
+  return (
+    <div {...css(styles.container)}>
+      <IndexLink
+        to="/"
+        {...css(styles.link), router.isActive('/') && styles.link_bold}
```

Comment:
Also, does this actually work? In the paragraph above, you mention that the IndexLink API is `activeClassName` and `activeStyle`.